### ISSUE TITLE
Drop `nwbinspector` as a dependency

### ISSUE THREAD

**Issue by h-mayorquin** [catalystneuro/nwb-conversion-tools#462]
The only place where we use it is here:
https://github.com/catalystneuro/nwb-conversion-tools/blob/7a52631a236f2be339e443401b0d2ca5162fec0b/src/nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py#L13

And that is a use of `check_regular_series` which is a super simple function:
https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dd58ddf34cfa0e1d81ec18739e1e3fa2f3e2cd8f/nwbinspector/utils.py#L45-L48

Are we planning on using more functionality? if not, I think that we should maintain a local version of the same function and drop the dependency as a whole.

**Comment by bendichter** [comment]
I agree, it doesn't seem worth it to have this dependency for such as simple function.

**Comment by CodyCBakerPhD** [comment]
The idea here wasn't just for the free utility function, which avoids duplicated code over multiple repo's, but also to encourage people to use the NWBInspector any time they are writing NWBFiles, which is the purpose of this repo.

**Comment by h-mayorquin** [comment]
I feel that right now we are saving the hypothetical user a pip install at the price of increasing our dependency complexity. My vote is for dropping it until code duplication becomes large enough that the scale is tipped or the need arises from another direction.
